plugins/node/opentelemetry-instrumentation-express/src/utils.ts

```diff
@@ -33,15 +33,21 @@ import {
  * @param request The request where
  * @param [value] the value to push into the array
  */
-export const storeLayerPath = (request: PatchedRequest, value?: string) => {
+export const storeLayerPath = (
+  request: PatchedRequest,
+  value?: string
+): { isLayerPathStored: boolean } => {
```

Comment:
This is extremely minor but I can't really think of any other information that could be returned by this, so personally I'd rather see this skip the object and just return a boolean, the meaning of which can be documented in the docstring.